### PR TITLE
Add plan schema for errata and minute plugins

### PR DESCRIPTION
**Pull request by lukaszachy**
Fixes: #1429

### CODE REVIEW

File: tmt/schemas/prepare/errata.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  how:
+    type: string
+    enum:
+      - errata
+
+  errata:
+    oneOf:
+      - anyOf:
+          - type: string
+            pattern: "^[0-9]+$"
```
Comment (happz) [inline tmt/schemas/prepare/errata.yaml:26]: Is leading zero really acceptable?

Comment (lukaszachy) [reply tmt/schemas/prepare/errata.yaml:26]: I see no reason why not, leading '0' should not make it problematic wrt int base 10 conversion any more. Easier to copy&paste from logs where one might find id filled with leading zeros.

Comment (happz) [reply tmt/schemas/prepare/errata.yaml:26]: Ok, makes sense, and it's true `int()` won't stumble.

**Comment by psss** [comment]
Why not dropping these files into the internal `tmt` packages together with the plugins themselves? The schemas are auto-picked so it would nicely work. Or am I missing anything?

**Comment by lukaszachy** [comment]
> Why not dropping these files into the internal `tmt` packages together with the plugins themselves? The schemas are auto-picked so it would nicely work. 

There are 3 ways how to detect plugins: drop-in to directory,  TMT_PLUGINS variable and  setup.py entry_point.
However schema has to be placed into the directory to be loaded, so we are missing way how to let tmt know about schema for plugins loaded by the other two way.

IMO at this point extending schema detection to support entry_point and TMT_PLUGINS is more work than benefit.

**Review by happz — APPROVED** [review]

**Review by psss — APPROVED** [review]
Yeah, agreed, as for now it seems the best way is to include these schemas here.

**Pull request merged**